wtxmgr/tx.go

```diff
@@ -173,68 +173,12 @@ func (s *Store) moveMinedTx(ns walletdb.ReadWriteBucket, rec *TxRecord, recKey,
 	log.Infof("Marking unconfirmed transaction %v mined in block %d",
 		&rec.Hash, block.Height)
 
-	// Insert block record as needed.
-	blockKey, blockVal := existsBlockRecord(ns, block.Height)
-	var err error
-	if blockVal == nil {
-		blockVal = valueBlockRecord(block, &rec.Hash)
-	} else {
-		blockVal, err = appendRawBlockRecord(blockVal, &rec.Hash)
-		if err != nil {
-			return err
-		}
-	}
-	err = putRawBlockRecord(ns, blockKey, blockVal)
-	if err != nil {
-		return err
-	}
-
-	err = putRawTxRecord(ns, recKey, recVal)
-	if err != nil {
-		return err
-	}
+	// Fetch the mined balance in case we need to update it.
```

Comment:
Is this change really just a refactor? Seems like a lot of code was removed, that I imagine did something 😛 

Maybe add a more descriptive commit message to make it easier to follow what this change is doing? :)

Reply:
It removes a lot of duplicate code between both methods that can be condensed into one.

Reply:
I see, so it changes the behaviour of `moveMinedTx`, but that's okay, since it is only used by `insertMinedTx`?

Reply:
Yep!